src/components/AdminDashboard/CompleteIncident.js

```diff
@@ -38,7 +37,7 @@ const CompleteIncident = props => {
   useEffect(() => {
     setFormValues({
       ...incident,
-      tags: incident.tags ? incident.tags.join(", ") : [],
+      tags: incident.tags ? incident.tags.join(', ') : [],
```

Comment:
The change to single quotes I would say is consistent since you did it across all of the javascript parts

Reply:
Interesting I just saw this on another PR and was wondering why. Well, as long as it works.